controllers/virtualmachinewebconsolerequest/webconsolerequest_suite_test.go

```diff
@@ -21,7 +21,7 @@ var intgFakeVMProvider = providerfake.NewVMProvider()
 
 var suite = builder.NewTestSuiteForControllerWithContext(
 	pkgcfg.NewContextWithDefaultConfig(),
-	v1alpha2.AddToManager,
+	virtualmachinewebconsolerequest.AddToManager,
```

Comment:
Will this work since you've changed to `addToManager`?

Reply:
Yep, the `addToManager` will be called from the `virtualmachinewebconsolerequest` package which has access to that function.